docs/adapters.rst

```diff
@@ -2,3 +2,5 @@ Adapters
 --------
 
 .. automodule:: openscm.adapters
+
+.. autoclass:: openscm.adapters.PH99
```

Comment:
no need for this, `automodule` already takes care of the classes
```suggestion
```

Reply:
Ah, sorry, my bad. It looks like Sphinx include includes those if they are part of `__all__` for that module (or directly declared in the file). Shall we use `__all__` here or the explicit inclusion of the class?